main.sh

```diff
@@ -27,27 +27,52 @@ Response=$(
     --url "$SERVER_URL/api-pub/v1/axe-watcher/gh/$COMMIT_SHA"
 )
 
-IssueCount=$(echo "$Response" | jq .last_run_violation_count)
+LastRunCreatedAt=$(echo "$Response" | jq -r .last_run_created_at)
+LastRunViolationCount=$(echo "$Response" | jq -r .last_run_violation_count)
+LastRunPageStateCount=$(echo "$Response" | jq -r .last_run_page_state_count)
+LastRunNewViolationCount=$(echo "$Response" | jq -r .last_run_new_violation_count)
+LastRunResolvedViolationCount=$(echo "$Response" | jq -r .last_run_resolved_violation_count)
 IssuesOverA11yThreshold=$(echo "$Response" | jq .issues_over_a11y_threshold)
 AxeURL=$(echo "$Response" | jq -r .axe_url)
 ProjectName=$(echo "$Response" | jq -r .project_name)
+DifferenceInPageStateCount=$(echo "$Response" | jq -r .difference_in_page_states)
 
 # Only set outs when running in GitHub Actions.
 if [ -n "${GITHUB_OUTPUT+x}" ]; then
   echo "project=$ProjectName" >>"$GITHUB_OUTPUT"
   echo "axe_url=$SERVER_URL$AxeURL" >>"$GITHUB_OUTPUT"
-  echo "issue_count=$IssueCount" >>"$GITHUB_OUTPUT"
+  echo "created_at=$LastRunCreatedAt" >>"$GITHUB_OUTPUT"
+  echo "issue_count=$LastRunViolationCount" >>"$GITHUB_OUTPUT"
+  echo "new_issues=$LastRunNewViolationCount" >> "$GITHUB_OUTPUT"
```

Comment:
`new_issues`, `resolved_issues`, `difference_in_page_states` are all only meaningful if there was a previous session for us to compare to. In the comparable summary UI we use for list items in the devhub frontend's "commits" view, we omit them if there wasn't a previous session.

I wonder if it might make more sense for us to update the walnut route to specify those properties as undefined or null if there wasn't a previous session, and to omit them here (from both `GITHUB_OUTPUT` and stdout) if so.

Reply:
I think that's a great idea. Would you like me to go ahead and work on modifying the output from the walnut route?

Reply:
I think that'd be good, but it doesn't have to block this PR